### PR TITLE
[HLS-2605] trace query with org

### PR DESCRIPTION
**Pull request by levyoren**

### CODE REVIEW

File: model/ids.go [hunk 1]
```diff
@@ -252,3 +255,21 @@ func (s *SpanID) UnmarshalJSON(data []byte) error {
...
+
+// ------- OrgId -------
+
+func NewOrgId(orgId uuid.UUID) OrgId {
+	return OrgId(orgId)
+}
+
+func (o OrgId) String() string {
+	return uuid.UUID(o).String()
+}
```
Comment (nata7che) [inline model/ids.go:268]: what is dis?

File: plugin/storage/badger/spanstore/reader.go [hunk 2]
```diff
@@ -172,6 +172,10 @@ func (r *TraceReader) GetTrace(ctx context.Context, traceID model.TraceID) (*mod
 	return nil, ErrInternalConsistencyError
 }
 
+func (r *TraceReader) GetTraceForOrg(ctx context.Context, traceID model.TraceID, orgId model.OrgId) (*model.Trace, error) {
+	return nil, nil
```
Comment (nata7che) [inline plugin/storage/badger/spanstore/reader.go:176]: should this return null?

Comment (nata7che) [reply plugin/storage/badger/spanstore/reader.go:176]: oh, is this because it's not implemented? maybe add a comment?

**Review by nata7che — APPROVED** [review]

File: model/ids.go (hunk 1 above)
Comment (levyoren) [reply model/ids.go:268]: a `toString()` method for the `OrgId` struct 🤦🏼‍♂️

File: plugin/storage/badger/spanstore/reader.go (hunk 2 above)
Comment (levyoren) [reply plugin/storage/badger/spanstore/reader.go:176]: will do.

**Pull request merged**